2.cellprofiler_analysis/scripts/nf1_analysis.py

```diff
@@ -8,40 +8,63 @@
 # In[1]:
 
 
+import sys
 import pathlib
 
-import sys
 sys.path.append("../")
 from utils import cp_utils
 
 
 # ## Set paths for each plate
+# 
+# Note: Due to the difference channel numbers between plates 1 + 2 (3 channels) and plates 3 + 3 prime (4 channels), there needs to be two difference cppipe files (like in the IC module). 
 
 # In[2]:
 
 
-path_to_output = pathlib.Path("./analysis_output").resolve(strict=True)
+path_to_output = pathlib.Path("./analysis_output").resolve()
```

Comment:
I do explain this in the module prior to this one (1.cellprofiler_ic), but I will add it to this one as well.

To answer here, this is the statement I have in the notebook/py file regarding the use of `strict`:

> Note: Output file path does not need to be strict since the `run_cellprofiler` function can create the output folder directory if it doesn't already exist. The other paths must be strict since these files should already exist for CellProfiler to run. The output directory doesn't need to already exist.